src/code-editor/settings-panel/settings-panel.ts

```diff
@@ -120,6 +121,25 @@ editor.once('load', () => {
     const fieldFormatOnSave = new BooleanInput();
     addField('Format On Save:', fieldFormatOnSave, 'ide.formatOnSave', 'If enabled the document will be auto-formatted on save');
 
+    // AI autocomplete settings
+    const fieldAutocompleteEnabled = new BooleanInput();
+    const groupAutocompleteEnabled = addField('Autocomplete:', fieldAutocompleteEnabled, 'editor.ai.autocompleteEnabled', 'Enable AI autocomplete suggestions while typing.');
```

Comment:
@isumygin-sc It seems that all Code Editor settings are store with prefix `ide.` but the autocomplete options have prefix `editor.ai.`. This PR doesn't change that....but since this is an unreleased feature, it might be a good idea to change this path.